geppetto-ui/src/components/menu/Menu.js

```diff
@@ -21,8 +21,8 @@ const menuStyle = {
       marginTop: '1px',
     },
     hover: {
-      background: "#11bffe",
-      backgroundColor: "#11bffe",
+      background: '#11bffe',
```

Comment:
Also these styles are not in line with the new styling principles but we have another pending pr and vfb is relying on these at the moment.